src/applications/vaos/covid-19-vaccine/components/SecondDosePage.jsx

```diff
@@ -1,23 +1,29 @@
 import React, { useEffect } from 'react';
 import { useHistory } from 'react-router-dom';
-import { connect } from 'react-redux';
-import * as actions from '../redux/actions';
+import { shallowEqual, useDispatch, useSelector } from 'react-redux';
 import FormButtons from '../../components/FormButtons';
 import { scrollAndFocus } from '../../utils/scrollAndFocus';
 import moment from 'moment';
-import { getReviewPage } from '../redux/selectors';
+import {
+  getCovid19VaccineFormPageInfo,
+  selectPageChangeInProgress,
+} from '../redux/selectors';
+import {
+  routeToNextAppointmentPage,
+  routeToPreviousAppointmentPage,
+} from '../redux/actions';
 
 const pageKey = 'secondDosePage';
 const pageTitle = 'When to expect a second dose';
 
-function SecondDosePage({
-  data,
-  facility,
-  pageChangeInProgress,
-  routeToNextAppointmentPage,
-  routeToPreviousAppointmentPage,
-}) {
+export default function SecondDosePage() {
+  const { data, facility } = useSelector(
+    state => getCovid19VaccineFormPageInfo(state, pageKey),
```

Comment:
This is a different selector from the one used in mapStateToProps in this component.

Reply:
Gotcha, yes I see the problem now